common/table.cpp

```diff
@@ -83,6 +83,48 @@ void Table::del(std::string key, std::string /* op */)
                            "DEL operation failed");
 }
 
+bool Table::getField(std::string key, std::string field, std::string &value)
+{
+    redisReply *reply = (redisReply*)redisCommand(
+            m_db->getContext(),
+            "HGET %s %s",
+            getKeyName(key).c_str(),
+            field.c_str());
```

Comment:
Please use redisFormatCommand. (See formatHSet API for exmaple)
'field' can be anything...
Please use RedisReply class as @stcheng commented.